src/common/linuxHelper/tests/linuxInfoHelper_test.cpp

```diff
@@ -22,7 +22,7 @@ TEST_F(LinuxInfoHelperTest, timeTick2unixTime)
     const auto startTimeNum {std::stoul(startTimeStr)};
     const auto startTimeUnixFunc {Utils::timeTick2unixTime(startTimeNum)};
 
-    const auto startTimeUnixCmd {std::stoul(Utils::Exec("date -d \"`ps -o lstart -p 1|tail -n 1`\" +%s"))};
+    const auto startTimeUnixCmd {std::stoul(Utils::Exec("LC_ALL=C date -d \"`ps -o lstart -p 1|tail -n 1`\" +%s"))};
```

Comment:
LC_ALL may have impact on other localization setting, can this have impact in other tests (new ones) after this?

Reply:
`LC_ALL=C` only impacts the command being executed, setting the language in which the response will be returned.